src/main/scala/org/apache/spark/ml/linalg/RAPIDSML.scala

```diff
@@ -73,6 +73,28 @@ private[spark] object RAPIDSML extends Serializable {
     jniRAPIDSML.dgemm(transa, transb, m, n, k, alpha, A.values, lda, B.values, ldb, beta, C.values, ldc, deviceID)
   }
 
+  /**
+   *
```

Comment:
There's no documentation as to what this does, just a list of parameter descriptions.

Reply:
Added.